src/features_creator/features_creator.py

```diff
@@ -38,3 +38,54 @@ def get_matching_column_names(data, pattern):
 
     """
     ...
+
+
+def calculate_percentage_change(
+    df, pattern, period=(2, 2), filter=None, changed_name=None
+):
+    """Calculate percentage change over a time period
+    (months over months or weeks over weeks)
+    for the given column pattern.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        A pandas dataframe
+    pattern : str
+        A column pattern:
+        pay_amt represents pay_amt1, pay_amt2...
+    period: tuple
+        Comparison period:
+        for 2 months over 2 months , period = (2, 2)
```

Comment:
Could you explain this a little bit? if I put period = (2, 3) or (4,2), what does that mean?